### DISCUSSION TITLE
播放视频无画面只有声音

### DISCUSSION THREAD

**Discussion by JunXiaoRuo** [AlistGo/alist#704] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/47996900/157169966-e255cafc-314b-4446-bec8-dc784970c385.png)

但是阿里云盘上面是正常的

![image](https://user-images.githubusercontent.com/47996900/157170020-675f3944-bf34-4736-80a5-83d765f2785c.png)

**Comment by JunXiaoRuo** [comment]
地址：http://pan.jrboy.cn/%E8%A7%86%E9%A2%91%E6%B5%8B%E8%AF%95/%E5%91%A8%E6%9D%B0%E4%BC%A6%EF%BC%9A%E9%AD%94%E5%A4%A9%E4%BC%A6%E4%B8%96%E7%95%8C%E5%B7%A1%E5%9B%9E%E6%BC%94%E5%94%B1%E4%BC%9A.mp4

**Comment by xhofe** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/36558727/157181479-62e476a6-ad70-4dd0-a977-0198f027123b.png)

**Comment by xifeng0** [comment]
我谷歌打开没画面，EDGE打开有

**Comment by orency** [comment]
这是h265或hevc编码的视频,目前主流浏览器里只有微软的新edge浏览器支持
